pkg/anago/release.go

```diff
@@ -419,4 +426,20 @@ func (d *DefaultRelease) CreateAnnouncement() error {
 	return nil
 }
 
-func (d *DefaultRelease) Archive() error { return nil }
+// Archive stores the release artifact in a bucket along with
+// its logs for long term conservation
+func (d *DefaultRelease) Archive() error {
+	// Create a new options set for the release archiver
+	archiverOptions := &release.ArchiverOptions{
+		ReleaseBuildDir: filepath.Join(workspaceDir, d.options.BuildVersion),
+		LogsDirectory:   filepath.Join(workspaceDir, "tmp"),
+		StageGCSPath:    "gs://" + filepath.Join(d.options.Bucket(), "stage"),
+		ArchiveGCSPath:  "gs://" + filepath.Join(d.options.Bucket(), "archive"),
+		BuildVersion:    d.options.BuildVersion,
+	}
```

Comment:
:warning:  I want to double check these values.

From the logs and the code, I think the paths inside the GCB workspace directory are different in `krel` than in `anago`. The values here are from the anago environment, let's talk about this.

Reply:
Do we even write the logs right now? We probably have to tell logrus to write to a file (beside to stderr). 

The workspace is hard-coded to `/workspace`, where everything else derives from it (like `gitRoot`).

Reply:
> Do we even write the logs right now? 

We do not write them, I thought about that and assumed we would write them at at some point to emulate what anago did. 

> The workspace is hard-coded to /workspace, where everything else derives from it (like gitRoot).

This is precisely what I wanted to confirm. From what I can understand, in anago, the build happened under a directory named with the version, for example gitRoot then was:

`/workspace/anago-v1.20.0-beta.2/src/`

Now, the version subdirectory is gone and the build happens one level up, directly in `/workspace`. So the git root is now just:

`/workspace/src/`

Am I right?

Reply:
Yep, that's right.

Created a PR for the logs https://github.com/kubernetes/release/pull/1774 (they will be put into $TMPDIR/{stage,release}.log and can be retrieved from the `state`)

Reply:
Awesome, I'll patch the archive step to collect them from the new names/locations.